frontend/device/pocketbook/device.lua

```diff
@@ -164,6 +173,20 @@ function PocketBook631:init()
     PocketBook.init(self)
 end
 
+function PocketBook626:init()
+    self.screen = require("ffi/framebuffer_mxcfb"):new{device = self, debug = logger.dbg}
+    self.powerd = require("device/pocketbook/powerd"):new{device = self}
+    self.input = require("device/input"):new{
+        device = self,
+        event_map = {
+            [24] = "LPgBack",
+            [25] = "LPgFwd",
+            [1002] = "Power",
+        }
+    }
+    PocketBook.init(self)
```

Comment:
Drat, I overlooked this before I merged. For future reference, please use `PocketBook:init()`. The `:` is so-called syntactic sugar that auto-passes self. :-)